leg_detector/CMakeLists.txt

```diff
@@ -39,6 +39,10 @@ include_directories(
   include ${catkin_INCLUDE_DIRS}
 )
 
+catkin_package(
+    INCLUDE_DIRS include
```

Comment:
since it doesn't not provide any libraries, I assume the headers are of very little use outside of this package. Hence, I'd say we leave them alone here and not export/install them.